test-suite/src/update.rs

```diff
@@ -0,0 +1,121 @@
+use {
+    crate::*,
+    gluesql_core::{
+        executor::{ExecuteError, UpdateError},
+        prelude::*,
+        translate::TranslateError,
+    },
+};
+
+test_case!(update, async move {
+    run!(
+        r#"
+CREATE TABLE TableA (
+    id INTEGER,
+    num INTEGER,
+    num2 INTEGER,
+    name TEXT,
+)"#
+    );
+    run!("INSERT INTO TableA (id, num, num2, name) VALUES (1, 2, 4, \"Hello\")");
+    run!("INSERT INTO TableA (id, num, num2, name) VALUES (1, 9, 5,\"World\")");
+    run!("INSERT INTO TableA (id, num, num2, name) VALUES (3, 4, 7,\"Great\")");
+    run!("INSERT INTO TableA (id, num, num2, name) VALUES (4, 7, 10,\"Job\")");
```

Comment:
1. You can merge these into a single `INSERT` query.
2. It would be useful to consider `r#"("test")"#` rust raw string literal representation. then you don't need to use `\"`

e.g.
```rust
run!(r#"
    INSERT INTO TableA (id, num, num2, name)
    VALUES
        (1, 2, 4, "Hello"),
        (1, 9, 5, "World"),
        (3, 4, 7, "Great"),
        (4, 7, 10, "Job");
"#);
```